src/components/calculator.js

```diff
@@ -1,19 +1,37 @@
 import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
```

Comment:
Performing operations on your app needs some improvements. Kindly ensure to resolve the following:
- [ ] After an operation is performed the results are used in the subsequent calculations. For instance, when I perform `2 + 1`, the result is `3`, and then if I next want to perform `4 + 5` it appears on the screen as `34 + 5` and yields a result of `39`, instead of `4 + 5` giving me `9`.
- [ ] The modulus (%) operator on your app does not perform any operations. When I perform `9 % 3` the screen only prints `93`.